task.sql

```diff
@@ -38,3 +38,18 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset mate.academy:5 labels:0.0.2
+CREATE TABLE Users (
+    ID INT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50),
+    PRIMARY KEY (ID)
+);
+--rollback DROP TABLE Users;
+
+--changeset mate.academy:6 labels:0.0.3
+CREATE INDEX Email ON Users (Email);
+--rollback ALTER TABLE Users DROP INDEX Email;
+ 
```

Comment:
You add whitespace in the last line and doesn't work EOF
![image](https://github.com/mate-academy/rda_task_9_running_database_migrations/assets/87720583/a71b3310-62df-471e-9b96-9374e1404ffe)